### PR TITLE
Hit the api

### PR DESCRIPTION
**Pull request by Ochiengsteven**
This pull request addresses the project requirements by integrating the Leaderboard API, creating a new game, and implementing the "Refresh" and "Submit" buttons for scores. Additionally, the codebase now utilizes arrow functions, async, and await to consume the API efficiently.

### Changes Made

- Integrated the Leaderboard API by reading its documentation to understand its functionalities and endpoints.

- Created a new game using the API, and saved the game ID returned by the API for future reference.

- Implemented the "Refresh" button, which fetches all the scores for the game created by the user from the API. The retrieved data is parsed as JSON and displayed to the user.

- Implemented the "Submit" button that allows users to save their scores for the game created by them. The relevant score data is sent to the API for storage and leaderboard update.

- Refactored existing code and replaced traditional functions with arrow functions to improve code readability and maintain consistency.

- Utilized async and await JavaScript features to consume the API asynchronously, enhancing overall performance and responsiveness.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -3,38 +3,82 @@ import renderPage from './modules/renderPg.js';
...
+  // get stored scores from the API
+  const getScores = async () => {
+    const response = await fetch(requestURL);
+    const scores = await response.json();
+    display(scores.result);
+  };
+
+  getScores();
+
+  function createScoreFromInputs() {
```
Comment (fobadara) [inline src/index.js]: [REQUIRED] Kindly change this to an arrow function. This is a requirement for this milestone.

File: src/index.js [hunk 2]
```diff
@@ -3,38 +3,82 @@ import renderPage from './modules/renderPg.js';
...
+    const inputName = document.getElementById('name-input').value;
+    const inputValue = document.getElementById('value-input').value;
+    createScore(inputName, inputValue);
+  }
+
+  function handleKeyPress(e) {
```
Comment (fobadara) [inline src/index.js]: [REQUIRED] Kindly change this to an arrow function. This is a requirement for this milestone.

File: src/index.js [hunk 3]
```diff
@@ -3,38 +3,82 @@ import renderPage from './modules/renderPg.js';
...
+    if (e.key === 'Enter') {
+      e.preventDefault();
+      createScoreFromInputs();
+    }
+  }
+
+  function handleSubmit(e) {
```
Comment (fobadara) [inline src/index.js]: [REQUIRED] Kindly change this to an arrow function. This is a requirement for this milestone.

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @Ochiengsteven, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Clean code
✅ Use of async and await

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 4]
```diff
@@ -3,38 +3,82 @@ import renderPage from './modules/renderPg.js';
...
+      method: 'POST',
+      body: JSON.stringify({
+        user: inputName,
+        score: inputValue,
+      }),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    });
+    // eslint-disable-next-line no-unused-vars
```
Comment (andersonlebon) [inline src/index.js:26]: ## [OPTIONAL]

- [ ]  Kindly make sure any linter checker is not disabled, they are really important as the help you structure your code and prevent future bugs by following  best practices, kindly solve them instead

File: src/index.js [hunk 5]
```diff
@@ -3,38 +3,82 @@ import renderPage from './modules/renderPg.js';
...
-    const cell = document.createElement('td');
-    cell.textContent = `${item.user}: ${item.score}`;
-    row.appendChild(cell);
-    tbody.appendChild(row);
-  });
-
-  table.appendChild(tbody);
-  scoresTable.appendChild(table);
+document.addEventListener('DOMContentLoaded', () => {
+  // eslint-disable-next-line no-unused-vars
```
Comment (andersonlebon) [inline src/index.js:7]: ## [OPTIONAL]

- [ ]  Kindly make sure any linter checker is not disabled, they are really important as the help you structure your code and prevent future bugs by following  best practices, kindly solve them instead

**Review by andersonlebon — APPROVED** [review]
## Hi  @Ochiengsteven 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**